ROADMAP.md

```diff
@@ -9,23 +9,9 @@ The following still needs work before this module can become a "1.0" release:
 * :book: Document foxxignore format in-app (`help foxxignore`?)
 * :book: Document foxxrc format in-app (`help foxxrc`?)
 
-## :x: Functionality
-
-### `init`
+## `init` (post-1.0)
 
 * Actually generate files and folders
 * Implement example routes (`generateExamples`)
 * Define meaningful "minimal" defaults
 * Define meaningful "full" defaults
-
-## :sparkles: Need pretty printing
-
-* `config`
-* `deps`
-* `install`
-* `replace`
-* `run`
-* `set-dev`
```

Comment:
We might want to add some output for `--verbose` to these two but I'm not sure about that.